x/txfees/keeper/txfee_filters/arb_tx.go

```diff
@@ -58,11 +58,27 @@ func isArbTxLooseAuthz(msg sdk.Msg, swapInDenom string, lpTypesSeen map[gammtype
 		}
 	}
 
+	multiSwapMsg, isMultiSwapMsg := msg.(poolmanagertypes.MultiSwapMsgRoute)
+	if isMultiSwapMsg {
+		for _, swapMsg := range multiSwapMsg.GetSwapMsgs() {
+			// TODO: Fix this later
```

Comment:
Can we clarify what the specific thing that needs to be fixed is?

Reply:
no

Reply:
`// drunk, fix later`